scenarioo-server/src/main/java/org/scenarioo/dao/diffViewer/GraphicsMagickConfiguration.java

```diff
@@ -23,26 +22,14 @@ private static boolean isGraphicsMagickAvailable() {
 		gmConsole.setOutputConsumer(new ArrayListOutputConsumer());
 		IMOperation gmOperation = new IMOperation();
 		try {
-			gmConsole.run(gmOperation);
+			gmConsole.run(gmOperation.version());
 		} catch (IOException e) {
-			// Ignore, we only want to know if graphics magick could be found
+			return false;
```

Comment:
@danielsuter I guess it might be a good idea to output some log in this case here, such that we have a chance on a installation to see in the log what went wrong (this is only loged once, which should be okay)